examples/osgviewerIPhone/iphoneViewerAppDelegate.mm

```diff
@@ -33,10 +41,19 @@ @implementation iphoneViewerAppDelegate
 
 @synthesize _window;
 
+//
```

Comment:
This change suggest to me that we need to change the osg::BuildShapeGeometryVisitor implementation so it does generate GL_QUADS to start with.  Could you post an this problem as an github Issue for the OSG to make sure it's not missed.